src/utils/clones.ts

```diff
@@ -163,7 +163,7 @@ function checkClonesPosition(
         nextSlide = childrenArr.length;
         nextPosition = -(itemWidth * nextSlide);
       } else {
-        nextSlide = totalItems - slidesToShow * 2;
+        nextSlide = totalItems / 3;
```

Comment:
Does this work for only 3 items or for all different cases with different item amounts?

Reply:
It's 3 sequences (copies) of items. Please take a look at my explanation in #296

Reply:
I'd suggest to introduce new constant for the sequences count, because I didn't found one while expecting the code. So for now it's just a hardcoded constant.

Reply:
And to answer to your question: yes, it will work for different amount in items. Already tested and used in production build of product that I maintain.

Reply:
And it makes sense that it'll work for all cases. Since the clones insert a copy of the original items before and after them. So the list of items is always a multiple of 3. 

`totalItems / 3` will always be the first item in the second bloc of clones.

Reply:
@YIZHUANG I'm not putting any pressure ;-) 
Just trying to figure out what you deployment cycle looks like.

Thanks